recipes/soci/all/conanfile.py

```diff
@@ -0,0 +1,154 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class SociConan(ConanFile):
+    name = "soci"
+    homepage = "https://github.com/SOCI/soci"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "The C++ Database Access Library "
+    topics = ("C++", "database-library", "oracle", "postgresql", "mysql", "odbc", "db2", "firebird", "sqlite3", "boost" )
```

Comment:
```suggestion
    topics = ("cpp", "database-library", "oracle", "postgresql", "mysql", "odbc", "db2", "firebird", "sqlite3", "boost" )
```

Avoid uppercase and symbols, it helps the search engine